assets/grayscale/app/views/grayscales/_footer.html.erb

```diff
@@ -0,0 +1,5 @@
+<footer>
+    <div class="container text-center">
+        <p>Copyright &copy; Your Website 2014</p>
```

Comment:
We don't have to hardcode the date since we're using Rails.  We can use `Time.now.year`